apps/opitx-md-editor/opitx.yml

```diff
@@ -0,0 +1,13 @@
+name: Opitx Markdown Editor
+description: 'A markdown editor for those tired of loosing their work'
+website: 'https://opitx-docs.netlify.com'
+repository: 'https://github.com/Thomashighbaugh/Opitx'
+keywords:
+    - React
+    - Markdown
+    - Note
+    - Electron
```

Comment:
Everything in the repo is an Electron app -- this keyword probably isn't necessary :slightly_smiling_face: